### DISCUSSION TITLE
安装不了

### DISCUSSION THREAD

**Discussion by 63490794** [subclipse/subclipse#165] [Q&A]
`Unable to connect to repository https://subclipse.github.io/u...
Connect to subclipse.github.io:443 [subclipse.github.io/127.0.0.1, subclipse.github.io/0:0:0:0:0:0:0:1] failed: Connection refused`

**Comment by markphip** [comment]
I do not really have any suggestions but all of those references to your loopback address are suspicious. It seems like something gave you the IP address of localhost instead of the GitHub site where things are hosted.

**Reply by 63490794** [reply]
好的，谢谢
